src/navigation/Tabs.js

```diff
@@ -13,51 +13,36 @@ import FontAwesome5 from 'react-native-vector-icons/FontAwesome5'
 export default function Tabs() {
-
     return (
         <NavigationContainer>
             <Tab.Navigator
-                screenOptions={({route})=>({
-                    tabBarIcon:({focused,size,color})=>{
-                        let iconName;
-                        if (route.name === 'Home'){
-                            iconName='home';
-                            size=focused? 25:20;
-                            color=focused? 'red':'#000';
-                        }else if (route.name === 'All books'){
-                            iconName='book';
-                            size=focused? 25:20;
-                            color=focused? 'red':'#000'
-                        }else if (route.name === 'List Of Books'){
-                            iconName='bookmark';
-                            size=focused? 25:20;
-                            color=focused? 'red':'#000'
-                        }else if (route.name === 'Book Shop'){
-                            iconName='shopping-basket';
-                            size=focused? 25:20;
-                            color=focused? 'red':'#000'
-                        }
-                        return(
-                            <FontAwesome5
-                                name={iconName}
-                                size={size}
-                                color={color}
-                            />
+                tabBarOptions={{ showLabel: false }}
+                screenOptions={({route}) => ({
+                    headerStyle: {backgroundColor: '#000'},
+                    headerTintColor: 'red',
+                    headerTitleAlign: 'center',
+                    tabBarIcon: ({focused}) => {
+                        const {iconName, size, color} = iconConfigs.getConfig(route.name, focused)
+                        return (
+                            <FontAwesome5 name={iconName} size={size} color={color}/>
                         )
-                    }
+                    },
+                    tabBarStyle: {
+                        backgroundColor: '#000',
+                        borderTopWidth: 0
+                    },
                 })
                 }
             >
-                <Tab.Screen name="Home" component={Home} options={{tabBarLabel: ()=> null}}/>
-                <Tab.Screen name="All books" component={AllBooks} options={{tabBarBadge: 3, tabBarLabel: ()=> null}}/>
-                <Tab.Screen name="List Of Books" component={ListOfBooksCurrently} options={{tabBarLabel: ()=> null}}/>
-                <Tab.Screen name="Book Shop" component={BookShop} options={{tabBarLabel: ()=> null}}/>
+                <Tab.Screen name="Home" component={Home}/>
+                <Tab.Screen name="All books" component={AllBooks} options={{tabBarBadge: 3}}/>
+                <Tab.Screen name="List Of Books" component={ListOfBooksCurrently}/>
+                <Tab.Screen name="Book Shop" component={BookShop}/>
```

Comment:
Screen name is a key, it should not contain spaces/special characters. `Book Shop` => `BookShop` => `Shop`